### DISCUSSION TITLE
Resize bicubic is not working for 3D image

### DISCUSSION THREAD

**Discussion by HussainAlasmawi** [Project-MONAI/MONAI#6609] [Q&A] [answered]
**Describe the bug**
monai.transforms.Resize with mode "bicubic" is not working for 3d images.

**To Reproduce**
Monai Version 1.2

Run commands
import monai
import torch

transform =  monai.transforms.Resize([224,224,224],mode='bicubic')
x = torch.ones([1,500,500,500])
transform(x)

**Screenshots**
![image](https://github.com/Project-MONAI/MONAI/assets/91467092/da61a144-8de2-4377-82b6-1b19b04c2110)

**Comment by KumoLiu** [comment, marked as answer]
Hi @HussainAlasmawi, the modes available for resizing are nearest, linear (3D-only), bilinear, bicubic (4D-only), trilinear (5D-only), area, and nearest-exact. For more information, you can refer to https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html.
Hope it can help you, thanks!
(convert to discussion for now)

**Reply by wyli** [reply]
(please see also a relevant feature request https://github.com/Project-MONAI/MONAI/issues/789)
